test/unit/query/autocomplete.js

```diff
@@ -27,8 +27,8 @@ module.exports.tests.query = function(test, common) {
   test('autocomplete + focus', function(t) {
     var query = generate({
       text: 'test',
-      lat: 29.49136,
-      lon: -82.50622
+      'focus.lat': 29.49136,
```

Comment:
yes, i'll fix it

On Wed, Sep 16, 2015 at 3:49 PM, Diana Shkolnikov <notifications@github.com>
wrote:

> In test/unit/query/autocomplete.js
> <https://github.com/pelias/api/pull/271#discussion_r39676270>:
>
> > @@ -27,8 +27,8 @@ module.exports.tests.query = function(test, common) {
> >    test('autocomplete + focus', function(t) {
> >      var query = generate({
> >        text: 'test',
> > -      lat: 29.49136,
> > -      lon: -82.50622
> > +      'focus.lat': 29.49136,
>
> should this be focus.point.lat?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pelias/api/pull/271/files#r39676270>.
>



-- 
Thanks,
Julian